index.html

```diff
@@ -0,0 +1,149 @@
+                    document.getElementById("calc").value = myOp;
+                } else {
+                    //overides previous operation in case user deviates from original selected operator
+                    op=myOp;
+                    document.getElementById("calc").value = myOp;
+                }
+            }
+        }
+
+        function calculate(){
+            if(val1 == undefined || op == undefined || isNaN(document.getElementById("calc").value)){
+                //do nothing if all variables are not defined or input value is not a number.
+                    //!!!Val2 is equal is in the input field at time of calculation
+            } else {
+                document.getElementById("calc").value = "...Loading answer";
+                val1 =undefined;
+                val2 =undefined;
+                op = undefined;
+            }
+        }
+        </script>
+
+
+        <div class="calculator buttons">
+
+            <button class="button command" id="clear" onclick="clearNum()">C</button>
+            <button class="button command" id="allClear" onclick="allClearNum()">AC</button>
+            <button class="button command" id="negative" onclick="makeNeg()">-/+</button>
+            <button class="button operator" id="divide"  onclick="myOperator('&#247;')">&#247;</button>
+
+            <button class="button number" id="numb" onclick="inputNum(7)">7</button>
+            <button class="button number" id="numb" onclick="inputNum(8)">8</button>
+            <button class="button number" id="numb" onclick="inputNum(9)">9</button>
+            <button class="button operator" id="multiply" onclick="myOperator('&#215;')">&#215;</button>
+
+            <button class="button number" id="numb" onclick="inputNum(4)">4</button>
+            <button class="button number" id="numb" onclick="inputNum(5)">5</button>
+            <button class="button number" id="numb" onclick="inputNum(6)">6</button>
+            <button class="button operator" id="subtract" onclick="myOperator('-')">-</but
+                ton>
+
+            <button class="button number" id="numb" onclick="inputNum(1)">1</button>
+            <button class="button number" id="numb" onclick="inputNum(2)">2</button>
+            <button class="button number" id="numb" onclick="inputNum(3)">3</button>
+            <button class="button operator" id="add" onclick="myOperator('+')">+</button>
+
+            <button class="button number zero" id="numb" onclick="inputNum(0)">0</button>
+            <button class="button number" id="decimal" onclick="decimal()">.</button>
+            <button class="button operator" id="equal" onclick="calculate()">=</button>
+
+        </div>
+        <!--[if lt IE 7]>
+            <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="#">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+        
+        <script src="" async defer></script>
+    </body>
+</html>
```

Comment:
Please make sure there is an empty line at the end of your files. This is considered a best practice for code files, especially ones that are generally concatenated together when bundled and interpreted, like in web development. 

What I mean by this is that if you have a bunch of html files, or JS files etc. in most projects these will be condensed down so that all the extra spaces we put in them to help us read are removed to make the files as small as possible. The line at the end of the file (EOF) helps determine where one file ends when these files are being bundled to avoid code bleeding into places it's not supposed to.